services/callback/src/main/java/app/coronawarn/server/services/callback/HashAlgorithmNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package app.coronawarn.server.services.callback;
+
+public class HashAlgorithmNotFoundException extends RuntimeException {
+
+  public HashAlgorithmNotFoundException(String msg) {
```

Comment:
why not including the root cause?
```suggestion
  public HashAlgorithmNotFoundException(String msg, Throwable rootCause) {
```